src/bundle/Resources/public/js/scripts/helpers/cookies.helper.js

```diff
@@ -0,0 +1,33 @@
+(function (global, doc, eZ) {
+    const setCookie = (name, value, expiryDays = 356, path = '/') => {
+        const date = new Date();
+        const expiryDaysInMiliseconds = expiryDays * 24 * 60 * 60 * 1000;
+
+        date.setTime(date.getTime() + expiryDaysInMiliseconds);
+        doc.cookie = `${name}=${value};expires=${date.toUTCString()};path=${path}`;
+    };
+    const getCookie = (name) => {
+        let cookieName = name + '=';
+        let decodedCookie = decodeURIComponent(doc.cookie);
+        let cookiesArray = decodedCookie.split(';');
+        let cookieValue = null;
+
+        cookiesArray.forEach((cookie) => {
+            cookie = cookie.trim();
```

Comment:
here is a lot of assignment for `cookie` which is function param, I don't think this is best practice.